derex/runner/mongodb.py

```diff
@@ -68,3 +116,33 @@ def copy_database(source_db_name: str, destination_db_name: str):
     cast(MongoClient, MONGODB_CLIENT).admin.command(
         "copydb", fromdb=source_db_name, todb=destination_db_name
     )
+
+
+@ensure_mongodb
+def create_root_user():
+    """Create the root user"""
+    create_user(MONGODB_ROOT_USER, MONGODB_ROOT_PASSWORD, ["root"])
+
+
+@ensure_mongodb
+def reset_mongodb_password(current_password: str = None):
+    """Reset the mongodb root user password"""
+    mongo_command_args = [
+        "mongo",
+        "--host",
+        "mongodb",
+        "--authenticationDatabase",
+        "admin",
+        "admin",
+        "--eval",
+        f'"db.changeUserPassword(\\"{MONGODB_ROOT_USER}\\",'
+        f'\\"{MONGODB_ROOT_PASSWORD}\\");"',
+    ]
+    if current_password:
+        mongo_command_args.extend(["-u", MONGODB_ROOT_USER, f"-p{current_password}"])
+
+    mongo_command = " ".join(mongo_command_args)
+    args = ["run", "--rm", "mongodb", "bash", "-c", f"{mongo_command}"]
```

Comment:
This will work only if the mongodb container is already running. We can speed things up by not creating a new container, using `exec` instead of `run`